src/app.js

```diff
@@ -56,11 +56,44 @@ function MonitorApp ({
   }
 
   async function getMonitorUpdates() {
```

Comment:
Might as well make this function name more specific while we're here. Could be something like your inner helper function -- `latestExitnodeStats()`. Or `numNodesLastSeenAtExitnodes()`.

Reply:
I ended up naming this to countActiveRoutesByExitnode(). Let me know if you like one of these other names better. Happy to change it to whatever